WikiDrips/WikiTableViewController.swift

```diff
@@ -79,13 +74,13 @@ class WikiTableViewController: UITableViewController, UISearchBarDelegate {
         
         let wikiDoc = wikiDocs[indexPath.section][indexPath.row]
         wikiCell.wikiTitleLabel?.text = wikiDoc.title
-        wikiCell.wikiDateLabel?.text = dateFormatter.string(from: wikiDoc.pubDate)
+        wikiCell.wikiDateLabel?.text = dateFormatter.string(from: wikiDoc.date)
         
         if let rectangle = wikiCell.wikiTitleImageView?.bounds {
             wikiCell.wikiTitleImageView?.image = UIImage.image(withInitials: wikiDoc.imageInitials, in: rectangle)
```

Comment:
@bartcull you can also do this in another ticket... the image generation was supposed to happen in background, not on main thread.